package.json

```diff
@@ -22,18 +22,20 @@
     "url": "https://github.com/bartaz/impress.js/issues"
   },
   "scripts": {
-    "lint": "jshint js/impress.js *.js test/bootstrap.js && jscs js/impress.js *.js test/bootstrap.js --preset=jquery",
+    "lint": "jshint js/impress.js test/*.js && jscs js/impress.js test/*.js",
     "test": "karma start --single-run",
     "test:dev": "karma start",
     "test:sauce": "karma start karma.conf-sauce.js"
   },
   "devDependencies": {
     "chrome": "0.1.0",
+    "firefox": "*",
```

Comment:
Is this dependency needed? Do we use firefox from node, or does karma requires it?

Reply:
Upstream only use Chrome for testing with karma. Well, at least in karma.conf.js. I'm actually unfamiliar with the Saucelabs setup.

I wanted to test with Firefox too, so I've added it. This module is needed, and there's a line in karma.conf.js that also adds Firefox to the list of browsers.

Note that Chrome doesn't allow JavaScript to access the inside of an iframe when the protocol is file:///. So for opening qunit_test_runner.html in a browser (without karma), it only works with Firefox. So for this reason it's actually important to at least keep Firefox in the CI loop. Chrome is then kinda optional, but I like testing with both and I guess I also wanted to keep Chrome as upstream has been using that. 

And to make it crystal clear: both of them work fine over karma, because then files are served over http.